### PR TITLE
Finished 06 Homework

### PR DESCRIPTION
**Pull request by rickymetz**
4

### CODE REVIEW

File: class06/rick/moma.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'pry'
+require 'active_record'
+
+ActiveRecord::Base.establish_connection('postgres://localhost/moma_db')
+
+ActiveRecord::Base.logger = Logger.new(STDOUT)
+
+class Style < ActiveRecord::Base
```
Comment (mobjohnson) [inline class06/rick/moma.rb]: This class is not required for this homework

File: class06/rick/moma.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS paintings;
+DROP TABLE IF EXISTS artists;
+DROP TABLE IF EXISTS nations;
+
+CREATE TABLE styles(
```
Comment (mobjohnson) [inline class06/rick/moma.sql]: also not needed

**Comment by mobjohnson** [comment]
you are missing the ar_commands.txt file that has your pry commands to perform Part 3.

You are welcome to submit a pull request that includes the ar_commands.txt, but I have marked you complete for this assignment.

File: class06/rick/moma.sql (hunk 2 above)
Comment (rickymetz) [inline class06/rick/moma.sql]: I know, I wanted to see if I could make an additional table as a challenge.

**Comment by rickymetz** [comment]
Whoops. I did on my local machine, but I didn't add them to a text file. I'll add it.

**Pull request merged**